.github/workflows/bifrost_http_release.yml

```
@@ -0,0 +1,208 @@
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Extract metadata for Docker
+        id: meta
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2034:warning:1:1: VERSION appears unused. Verify use (or export if used externally)

(shellcheck)